src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1894,9 +1873,6 @@ limitations under the License.
                 <requireAttribute>CLIENT_LIST</requireAttribute>
                 <requireAttribute>PARTS_LIST</requireAttribute>
             </include>
-            <include cluster="Binding" client="false" server="false" clientLocked="true" serverLocked="false">
```

Comment:
The All-clusters-app device (not in the spec) should have Binding server, no?

Reply:
It is not defined, do we need to to enable any client cluster on the  all-clusters-app? I think might be OTA Software Update Provider.

Maybe we should left as is for flexibility. 
<include cluster="Binding" client="false" server="false" clientLocked="true" serverLocked="false">

Reply:
all-cluster-app has enabled OTA Software Update Provider with client role by default, so I include Binding in all-cluster-app with default on and allow the user to turn off.

`<include cluster="Binding" client="false" server="false" clientLocked="true" serverLocked="false">`